06week/Checkpoint2.js

```diff
@@ -0,0 +1,105 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = = strNums.split(',').map(function(item) {
+    return parseInt(item, 10);
+});// Must split the array before turning the strings into numbers.
+
+
+// Find the sum of the even values
+const sumEvens = strNums.reduce((total, num) =>{
+if(nums % 2==0){
+  return total + num;
+})// Use reduce to get a single value from the array if the nums are even.
+
+console.log(sumEvens);
+
+// Find the index of the first "Austin" value (the value plus its index equals 512)
+const atxIdx = nums.find((item, index))=>{
+  item + index == 512;
```

Comment:
Don't forget to return.  Also, you have the closing ")" after index, ending the callback before executing.